docs/current_docs/api/arguments.mdx

```diff
@@ -356,8 +436,25 @@ Here is an example of passing a local directory to this Dagger Function as argum
 ```shell
 mkdir -p mydir/mysubdir
 touch mydir/a mydir/b mydir/c mydir/mysubdir/y mydir/mysubdir/z
+```
+
+<Tabs groupId="shell">
+<TabItem value="System shell">
+```shell
+dagger -c 'tree mydir 2'
```

Comment:
Just a (general) idea for another time, but the `depth` argument could default to 1 so it becomes optional. Then this would become:
```
tree mydir --depth=2
```
This can alleviate concerns about clarity on what arguments mean.

Reply:
Yes, good idea!

Reply:
Opened https://github.com/dagger/dagger/pull/9923 for this but closed it because I realized that at this point in the page, we have not yet introduced default args.